CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 
 cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
 
-project(icaruscode VERSION 09.42.04 LANGUAGES CXX)
+project(icaruscode VERSION 09.43.00 LANGUAGES CXX)
```

Comment:
I am not sure if this is not reviewed/approved by others by tomorrow these versions will have to be updated to be merged next week. Version will change tomorrow (Friday)with weekly release